posix.mak

```diff
@@ -497,6 +498,27 @@ ${DUB}: ${DUB_DIR} ${STABLE_DMD}
 	cd ${DUB_DIR} && DC="$(abspath ${STABLE_DMD}) -conf=$(abspath ${STABLE_DMD_CONF})" ./build.sh
 
 ################################################################################
+# HTML hyphenation
+################################################################################
+
+# binary
+.PHONY: html-hyphenator
+html-hyphenator: ${DUB} ${STABLE_DMD}
+	DFLAGS="$(DPL_DOCS_DFLAGS)" ${DUB} build --root=${HTML_HYPHENATOR_PATH} \
+		--compiler=${STABLE_DMD}
+
+# run hyphenator (ignore dpl-docs which has integrated hyphenation support)
+# hyphenate any html file that is newer than the rule's .done dummy file
+# docs dependencies are order-only so it's possible to hyphenate only some of them
+${HTML_HYPHENATOR_PATH}/.done: ${ALL_FILES} html-hyphenator \
+		| phobos-prerelease druntime-prerelease druntime-release phobos-release
```

Comment:
Shouldn't a dependency here be also on html? You need the html (of the site proper) done before you do the hyphenation.